From b29b47c0c89d33306227556f7fe3aee620e0e47c Mon Sep 17 00:00:00 2001
From: Paul Kocialkowski <contact@paulk.fr>
Date: Fri, 21 Apr 2017 13:31:56 +0200
Subject: [PATCH 1/4] udev: Add name-based input device detection without dmi

Some devices do not use dmi at all (this is the case on most non-x86
platforms, such as arm and arm64) but should able to select specific
quirks based on the input device name too.

This adds name-based input device detection without dmi to support
these devices.

Signed-off-by: Paul Kocialkowski <contact at paulk.fr>
---
 udev/90-libinput-model-quirks.rules.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/udev/90-libinput-model-quirks.rules.in b/udev/90-libinput-model-quirks.rules.in
index 8bff192..149ef0d 100644
--- a/udev/90-libinput-model-quirks.rules.in
+++ b/udev/90-libinput-model-quirks.rules.in
@@ -37,4 +37,8 @@ ENV{ID_INPUT_MOUSE}=="1", \
 KERNELS=="input*", \
   IMPORT{builtin}="hwdb 'libinput:name:$attr{name}:$attr{[dmi/id]modalias}'"
 
+# libinput:name:<name>
+KERNELS=="input*", \
+  IMPORT{builtin}="hwdb 'libinput:name:$attr{name}'"
+
 LABEL="libinput_model_quirks_end"
-- 
2.12.2

